src/ui/editortabwidget.cpp

```diff
@@ -197,6 +216,7 @@ void EditorTabWidget::tabRemoved(int)
 
 Editor *EditorTabWidget::currentEditor()
 {
+    setFocusProxy(editor(currentIndex()));
```

Comment:
I just removed it and it broke things again. Guess it's needed at least to some degree.